client/src/components/TitleInputModal.js

```diff
@@ -12,26 +12,14 @@ import { makeStyles } from "@material-ui/core/styles";
 import CloseIcon from "@material-ui/icons/Close";
```

Comment:
<img width="1084" alt="Screen Shot 2020-03-02 at 2 30 34 PM" src="https://user-images.githubusercontent.com/37638884/75710348-6c4a2300-5c92-11ea-9a68-9a03e1923172.png">
if I add a value to  title in column modal and then close the modal, when i re-open the modal. the value is still there. also If i press create card without a title and close the modal, and then reopen the error is still there. please try to fix these by setting title to blank and error to false after close. Also the console error shows theres something wrong with how youre handling the input. I would find out the issue and fix that

Reply:
also make sure these fixes are applied to all other modals that use this component like create board

Reply:
I changed code. please check it.